Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian
+
+RUN apt update && apt install -y curl
+RUN curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
+RUN apt-get install -y nodejs
+
+# NOTE: We are only installing chromium to get its dependencies
+RUN apt install -y chromium
```

Comment:
What dependencies exactly, and won't those cause any issue by using an older chromium version?

i.e. given puppeteer v5.3.1 comes with Chromium 86

Reply:
> What dependecies exactly

On Linux the Chromium package depends on a whole bunch of OS libraries. Specifically:
```
# apt-cache depends chromium
chromium
  Depends: libasound2
  Depends: libatk-bridge2.0-0
  Depends: libatk1.0-0
  Depends: libatomic1
  Depends: libatspi2.0-0
  Depends: libc6
  Depends: libcairo2
  Depends: libcups2
  Depends: libdbus-1-3
  Depends: libdrm2
  Depends: libevent-2.1-7
  Depends: libexpat1
  Depends: libflac8
  Depends: libfontconfig1
  Depends: libfreetype6
  Depends: libgbm1
  Depends: libgcc-s1
  Depends: libglib2.0-0
  Depends: libjpeg62-turbo
  Depends: libjsoncpp24
  Depends: liblcms2-2
  Depends: libminizip1
  Depends: libnspr4
  Depends: libnss3
  Depends: libopenjp2-7
  Depends: libopus0
  Depends: libpango-1.0-0
  Depends: libpng16-16
  Depends: libpulse0
  Depends: libre2-9
  Depends: libsnappy1v5
  Depends: libstdc++6
  Depends: libwebp6
  Depends: libwebpdemux2
  Depends: libwebpmux3
  Depends: libx11-6
  Depends: libxcb1
  Depends: libxcomposite1
  Depends: libxdamage1
  Depends: libxext6
  Depends: libxfixes3
  Depends: libxkbcommon0
  Depends: libxml2
  Depends: libxrandr2
  Depends: libxslt1.1
  Depends: zlib1g
```
The Chromium shipped with puppeteer doesn't install these (assumes that these are there) and fails to launch if they are missing in the docker container. 

> won't those cause any issue by using an older chromium version?

From the list above as you can see these provide fairly standard functionality, mostly around parsing image formats and XML etc... they are fairly stable and while there might have been changes from version 86 to 103, I have tested running the puppeteer code and can confirm that Chromium 86 launches successfully with these dependencies installed.

I'll replace the above "install chromium" with "install this list of packages".